.github/ISSUE_TEMPLATE/question.yml

```diff
@@ -0,0 +1,23 @@
+name: Question
```

Comment:
We can probably just use the links in the template types now to point directly to our discussion page and SO instead of having anything here at all? Thoughts?

Reply:
So 2 separate links, "Questions & Help" and "Ask on StackOverflow", and it would effectively remove the ability to ask questions in issue format. 

I see no problems with this :)

Reply:
That's true... what about all 3? Too much?

Reply:
Depends on if you want to allow future questions to be posted as issues.

Questions & Help seems much more appropriate, and they'll always have the option to use the blank issue template at the bottom.